pinot-tools/src/main/resources/examples/stream/meetupRsvp/complexTypeHandling_meetupRsvp_realtime_table_config.json

```diff
@@ -26,23 +26,13 @@
       ]
     },
     "transformConfigs": [
-      {
-        "columnName": "group_json",
-        "transformFunction": "jsonFormat(\"group\")"
-      }
     ],
     "complexTypeConfig": {
       "unnestFields": ["group.group_topics"]
     }
```

Comment:
Now that we have schema inference and can see the fully exploded schema after unnesting, can we add a test to unnest a collection which also has an inner collection. I think we added a TODO for the same in previous PR ?

Reply:
yes, that's why there are more columns added to the schema file to include all the nested columns (which are inferred using schema reference)

Reply:
Right. So group_topics is the collection we are unnesting and inside this collection I don't think there is another collection. So if we put some dummy data to create another array inside group_topics, then we can add the following unnest config to the test 

```
"complexTypeConfig": {      
   "unnestFields": [
       "group.group_topics",
       "group.group_topics.inner_array"
    ]   
 }
```

I don't think unnesting collection inside another collection is covered in other tests.

Reply:
oh, you want inner nested collection. No, the quick start sources that we have do not include such a case.

Reply:
But the handling is covered by unit tests.

Reply:
Yes wanted that in quick start just so user can get a demo for deep unnesting. We can probably add it later in the quickstart.